mmv1/third_party/terraform/website/docs/guides/using_write_only_attributes.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+page_title: "Use write-only attributes in the Google Cloud provider"
+description: |-
+  How to use write-only attributes in the Google Cloud provider
+---
+
+# Write-only attributes in the Google Cloud provider
+
+Write-only attributes allow users to access and use data in their configurations without that data being stored in Terraform state.
+
+Write-only attributes are available in Terraform v1.11 and later. For more information, see the [official HashiCorp documentation for Write-only Attributes](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
+
+To mark the launch of write-only attributes, the Google Cloud provider has added the following write-only attributes:
```

Comment:
Can this text be made evergreen? This reads as temporary- like an announcement blog post- but will be used by users as evergreen content.